reference/wishlists.v3.yml

```diff
@@ -354,7 +363,7 @@ paths:
               type:
                 type: string
     delete:
-      description: Deletes a Wishlist.
+      description: Deletes a wishlist.
       summary: Delete a Wishlist
```

Comment:
```suggestion
      summary: Delete a wishlist
```

Reply:
Changing the summary field will change the way the name of the endpoint appears in the left pane of API Reference.